src/assets/JSAsset.js

```diff
@@ -102,14 +100,15 @@ class JSAsset extends Asset {
   }
 
   async parse(code) {
-    const options = await this.getParserOptions();
-
-    if (options.babelVersion === 6) {
-      let babylon = await localRequire('babylon', this.name);
-      return babylon.parse(code, options);
-    } else {
-      return babelCore.parse(code, options);
+    // const options = await this.getParserOptions();
+    // return babelCore.parse(code, options);
+    await babel(this);
```

Comment:
Moved babel into parse. Its the first thing to need the AST, so makes sense for parsing to happen as part of babel's normal processing. If babel doesn't get applied to that file, then we fall back to parsing with `@babel/parser`.